torch/csrc/jit/runtime/interpreter.cpp

```diff
@@ -1640,8 +1640,8 @@ struct InterpreterStateImpl : c10::intrusive_ptr_target {
       Node* node = frame.function->instructions_source_[pc];
       if (node->callstack()) {
         for (const auto& p : (*node->callstack())->vec()) {
-          entries.emplace_back(StackEntry{previous_fn_name, p.second});
-          previous_fn_name = p.first->name();
+          entries.emplace_back(StackEntry{previous_fn_name, std::get<1>(p)});
```

Comment:
@ZolotukhinM, are you ok with these changes? It seems from last time you commented on the fact that InlinedCallStack's vec method has no other users, to now it has changed. And it seems it is being used here. Do you prefer this change or add another API to InlinedCallStack?

Reply:
This change looks good to me!